### ISSUE TITLE
AttributeError: 'Adam' object has no attribute 'learning_rate'

### ISSUE THREAD

**Issue by wubowen416** [surmenok/keras_lr_finder#24]
Thank you for this repo!

I saw that you rewrite the "lr" to "learning_rate" but now new problems appears..

This is my code
```
model.compile(loss=scaled_loss,
              optimizer='adam')

lr_finder = LRFinder(model)
lr_finder.find(X_train, Y_train, 1e-6, 1e-2, 128, 5)
```

And the exception is 
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-14-a360a4ec21ae> in <module>
     17 
     18 lr_finder = LRFinder(model)
---> 19 lr_finder.find(X_train, Y_train, 1e-6, 1e-2, 128, 5)

~/anaconda3/envs/myenv/lib/python3.7/site-packages/keras_lr_finder-0.1-py3.7.egg/keras_lr_finder/lr_finder.py in find(self, x_train, y_train, start_lr, end_lr, batch_size, epochs)
     52 
     53         # Remember the original learning rate
---> 54         original_lr = K.get_value(self.model.optimizer.learning_rate)
     55 
     56         # Set the initial learning rate

AttributeError: 'Adam' object has no attribute 'learning_rate'
```

The attribute of learning rate in my case is 'lr'.
However when I take a look at keras doc, it is written as
```
class Adam(Optimizer):

    def __init__(self, learning_rate=0.001, beta_1=0.9, beta_2=0.999,
                 amsgrad=False, **kwargs):
        self.initial_decay = kwargs.pop('decay', 0.0)
        self.epsilon = kwargs.pop('epsilon', K.epsilon())
        learning_rate = kwargs.pop('lr', learning_rate)
        super(Adam, self).__init__(**kwargs)
        with K.name_scope(self.__class__.__name__):
            self.iterations = K.variable(0, dtype='int64', name='iterations')
            self.learning_rate = K.variable(learning_rate, name='learning_rate')
            self.beta_1 = K.variable(beta_1, name='beta_1')
            self.beta_2 = K.variable(beta_2, name='beta_2')
            self.decay = K.variable(self.initial_decay, name='decay')
        self.amsgrad = amsgrad
```
It is **self.learning_rate = K.variable(learning_rate, name='learning_rate')**.

I don't know why this is happening. Even not sure whether should the issue be opened in this repo.

But anyway, I think it maybe better if both 'lr' and 'learning_rate' could be handled.

**Comment by wubowen416** [comment]
I changed the name of variable of keras in my env...

**Comment by sushant3095** [comment]
how did you do that?

> I changed the name of variable of keras in my env...

**Comment by wubowen416** [comment]
> how did you do that?
> 
> > I changed the name of variable of keras in my env...

I just directed to Keras .py files (optimizers.py I guess) and found the Adam class. Then I realized that in my Keras the attribute name is ‘lr’. So I rewrites it to ‘learning_rate’.

However I don’t recommend you to do that since other file will raise error due to this modification.

So better to rewrite the index for taking learning rate of this repo I guess.
